RealmSwift/EmbeddedObject.swift

```diff
@@ -165,62 +165,6 @@ extension EmbeddedObject: RealmCollectionValue {
         return RLMDynamicGet(self, prop)
     }
 
-    // MARK: Notifications
```

Comment:
There doesn't seem to be a clean way to do this from an extension.

Reply:
The public thing needs to be on Object/EmbeddedObject which then could call an internal function on the ObjectBase extension rather than putting the public thing on ObjectBase.